t/03-dns-client/02-timer-usage.t

```diff
@@ -1,78 +0,0 @@
-use Test::Nginx::Socket;
-
-plan tests => repeat_each() * (blocks() * 5);
-
-workers(6);
-
-no_shuffle();
-run_tests();
-
-__DATA__
-
-=== TEST 1: reuse timers for queries of same name, independent on # of workers
```

Comment:
Because `02-timer-usage.t` did not test the scenario when stale data triggerred the async timer. It only tested async query logic in syncQuery().

If we want to test stale data triggered async logic, we need to write a new one.

Reply:
hi @Tieske I added a new test case to check the scenario where the stale data triggers async timer